### ISSUE TITLE
Outdated `use-latest-callback` dependency causes type error

### ISSUE THREAD

**Issue by rsmidt** [callstack/react-native-paper#4579]
### Current behaviour
When trying to use a simple `FAB.Group`, a type error is emitted because of the "use-latest-callback" dependency:

```
Warning: TypeError: 0, _useLatestCallback.default is not a function (it is undefined)

This error is located at:
    in CardComponent (created by FAB.Group)
    in RCTView (created by View)
    in View (created by FAB.Group)
    in RCTView (created by View)
    in View (created by FAB.Group)
    in RCTView (created by View)
    in View (created by FAB.Group)
    in RCTView (created by View)
    in View (created by FAB.Group)
    in RCTView (created by View)
    in View (created by FAB.Group)
    in FAB.Group (created by TeamActionsFab)
....
```

Seems like a simple dep 

### Expected behaviour
It should not throw a type error.

### How to reproduce?
The component definition:
```tsx
function TeamActionsFab({ teamId }: TeamActionsFabProps) {
  const [fabState, setFabState] = useState({ open: false });

  const onStateChange = ({ open }: { open: boolean }) => setFabState({ open });

  return (
    <Portal>
      <FAB.Group
        visible
        open={fabState.open}
        icon="plus"
        actions={[
          {
            icon: "calendar",
            label: i18n.t("app.teams.actions.add_event"),
            onPress: () => console.log("Pressed add event"),
          },
        ]}
        onStateChange={onStateChange}
      />
    </Portal>
  );
}
```

### What have you tried so far?
There's a similar issue in react-navigation: https://github.com/react-navigation/react-navigation/issues/12056.
I've tried the steps laid out there (changing resolution, manually installing a newer version, ...), but nothing worked out.

**What actually worked** is manually overriding the dependency in the package.json:
```json
  "overrides": {
    "use-latest-callback": "^0.2.3"
  }
```

If it's just a matter of bumping the version, I can prepare a PR.

### Your Environment

| software                      | version
| --------------------- | -------
| ios                                | x (not used yet)
| android                        | 35
| react-native                | 0.76.5
| react-native-paper    | 5.12.5
| node                            | 22.11.0
| npm or yarn                | npm  (10.9.0)
| expo sdk                     | 52.0.19

**Comment by BogiKay** [comment]
Hey @rsmidt, this issue is not related to the paper package itself, but it may happening because npm resolves packages in different way if major version is 0, so I'll update dep to `0.2` to prevent using broken version.

**Comment by shahzaib-cmyk** [comment]
Did anyone have any luck with this? I'm not able to use react native paper v5 because of this exact bug in my app. I use expo router. Even the overrides key didn't help although I added the key only and didn't have the package installed separately when I did it.

**Comment by shahzaib-cmyk** [comment]
For anyone using pnpm, you can set it like this
`"pnpm": {
    "overrides": {
      "use-latest-callback": "^0.2.3"
    }
}`
This will go in you projects package.json, or the root package.json if using a monorepo like I was.
Other than that this seems like a pretty big issue. Hoping it's solved soon.

**Comment by daniel112** [comment]
> Hey [@rsmidt](https://github.com/rsmidt), this issue is not related to the paper package itself, but it may happening because npm resolves packages in different way if major version is 0, so I'll update dep to `0.2` to prevent using broken version.

when is this new update going live? it is still 0.1.5 on the main branch?

**Comment by SasquatchLV** [comment]
Yeah, same issue. Just updated react native to 0.79 and this came up - TypeError: 0, _useLatestCallback.default is not a function (it is undefined) 

Specifically points to node_modules/react-native-paper/src/components/Card/Card.tsx

**Comment by M1chael30** [comment]
how to fix this error? i don't know how to fix this

**Comment by Roalkege** [comment]
~~Installing `npm i @react-native-aria/utils` solved it for me.~~

Don't do it, it contains malicious code.

**Comment by zohaib-ahmad1020** [comment]
localhost:8081/index…pp.testing.uat:1170 Warning: TypeError: 0, _useLatestCallback.default is not a function (it is undefined)

This error is located at:
    at Chip (http://localhost:8081/index.bundle//&platform=ios&dev=true&lazy=true&minify=false&inlineSourceMap=false&modulesOnly=false&runModule=true&excludeSource=true&sourcePaths=url-server&app=com.adgm.app.testing.uat:229907:25)

**Comment by conor909** [comment]
expo 53 seems to have issues, [brentvatne](https://github.com/brentvatne)'s comment fixed this for me here:
https://github.com/expo/expo/issues/36477

**Comment by abitling** [comment]
Hey @zohaib-ahmad1020 @SasquatchLV, I encountered the same issue after upgrading React Native from `0.74.3` to `0.79.2`.

I initially tried several suggested workarounds—such as adding forced resolutions in `package.json` and setting `config.resolver.unstable_enablePackageExports = false` in the Metro config—but unfortunately, none of them resolved the issue.

Eventually, I decided to generate the JS bundle manually and inspect it closely. That’s when I discovered the root cause: I had applied custom patches to `react-native-paper`, specifically modifying `lib/commonjs/components/Modal.js` and its corresponding source map file.

The problem stemmed from patched code that used the `_useLatestCallback` function. During minification, this function appears to be renamed or stripped, which led to the runtime errors I was seeing.

Hope this helps anyone else facing similar symptoms!

**Comment by Mukesh-Murugaiyan** [comment]
Warning: TypeError: 0, _useLatestCallback.default is not a function (it is undefined)

"react-native-tab-view": "^4.1.2",
"react-native-pager-view": "^6.8.1",
"react-native-gesture-handler": "^2.27.1",
"react-native-reanimated": "~3.17.4",
"react-native-screens": "~4.11.1"
These versions are fully compatible with the latest Expo and React Native versions as of writing.

🛠️ Run the App
Once you’ve installed the above dependencies, run the following commands to ensure a clean start:

npx react-native start --reset-cache
npx react-native run-android
The --reset-cache option ensures that any previous cache is cleared, which can help resolve common issues with Metro bundler and ensure smooth development.

This setup worked perfectly for me, and I hope it helps you too! If you're facing issues with tab navigation in newer versions, give this combination a try.

**Comment by n8b8dy** [comment]
@Roalkege, unfortunately, this library is deprecated and contains [malicious code](https://osv.dev/vulnerability/MAL-2025-4792).

**Comment by Roalkege** [comment]
> @Roalkege, unfortunately, this library is deprecated and contains [malicious code](https://osv.dev/vulnerability/MAL-2025-4792).

Only v0.2.13. This version isn't available. But I added it to my comment.

**Comment by geoffcfchen** [comment]
```
 "overrides": {
    "use-latest-callback": "^0.2.3"
  }
```

does work for expo 53.
